frontend/src/components/ui/ExecutionDataGrid.jsx

```diff
@@ -34,30 +35,33 @@ export const PipelineTableRow = ({ row, getRowProps }) => {
 };
 
 export const ExecutionDataGrid = ({ closeModal }) => {
-  const [, setWorkspace] = useImmerAtom(workspaceAtom);
   const [lineage] = useAtom(lineageAtom);
   const [currentPage, setCurrentPage] = useState(1);
   const [pageSize, setPageSize] = useState(15);
   const syncResults = useSyncExecutionResults();
   const [configuration] = useAtom(activeConfigurationAtom);
   const loadExecution = useLoadExecution();
+  const queryClient = useQueryClient();
+  const { addPipeline } = useWorkspace();
+
+  useEffect(() => {
+    if (configuration?.anvil?.host) {
+      queryClient.invalidateQueries({
+        queryKey: ["pipelines", configuration?.anvil?.host],
+      });
+    }
+  }, [configuration?.anvil?.host]);
```

Comment:
Do we need to a `useEffect` for this? Can we check if the anvil host changed since the last render? 
https://react.dev/learn/you-might-not-need-an-effect#adjusting-some-state-when-a-prop-changes

Reply:
You're right, this doesn't even do anything, removed it